x-pack/plugins/alerting/server/invalidate_pending_api_keys/task.ts

```diff
@@ -124,42 +112,22 @@ function taskRunner(
       async run() {
         let totalInvalidated = 0;
         try {
-          const [{ savedObjects, http }, { encryptedSavedObjects, security }] =
-            await coreStartServices;
-          const savedObjectsClient = savedObjects.getScopedClient(
-            getFakeKibanaRequest(http.basePath.serverBasePath),
-            {
-              includedHiddenTypes: ['api_key_pending_invalidation'],
-              excludedExtensions: [SECURITY_EXTENSION_ID],
-            }
-          );
+          const [{ savedObjects }, { encryptedSavedObjects, security }] = await coreStartServices;
+          const savedObjectsClient = savedObjects.createInternalRepository([
```

Comment:
The `ad_hoc_run_param` SO are space specific, so I want to use `namespaces: ['*']` to search across all namespaces. Using `getScopedClient` with a fake request did not provide enough privileges to do this so I am switching to use `createInternalRepository` which I believe is ok since this is a system task and shouldn't be tied to any (fake) user privileges.